demos/demo.tmux.exec.sh

```
@@ -0,0 +1,117 @@
+
+# --- Argument Parsing ---
+arguments::init \
+  --timestamp "Add timestamp to each output line" \
+  --line-number "Add line numbers to output" \
+  --help "Show help message" \
+  -- "SCRIPT" "Script or command to execute (required)"
+
+arguments::parse "$@"
+
+if arguments::has --help; then
+  echo "\n${cl_cyan}Usage:${cl_reset} $(basename "$0") [--timestamp] [--line-number] <script_or_command> [args...]\n"
```

Shellcheck (0.10.0)

[warning] cl_cyan is referenced but not assigned.

(SC2154)